pages/common/mysqld.md

```diff
@@ -0,0 +1,19 @@
+# mysqld
+
+> The MySQL Server.
+
+- Display error log messages:
```

Comment:
I think you should also mention `Start the server`, since this is the first example and it should be clear that the mysqld command starts up a mysql server.